apps/builder/app/routes/rest.current.products.ts

```diff
@@ -1,71 +0,0 @@
-import { json, type LoaderFunctionArgs } from "@remix-run/server-runtime";
```

Comment:
We don't use it in n8n?

Reply:
decided to trust this comment
/**
 * Created for ebugging purposes, to check that payments and products are working
 * Showing current user's checkouts with products.
 */